app/views/form/guidance/_finding_scheme.erb

```diff
@@ -0,0 +1,5 @@
+<%= govuk_details(summary_text: "Can’t find your scheme?") do %>
+  <p class="govuk-body">If your organisation’s schemes were migrated from old CORE, they may have new names and codes. Search by postcode to find your scheme.</p>
+  <p class="govuk-body"><%= govuk_link_to("View your organisation’s schemes.", schemes_path) %></p>
+  <p class="govuk-body"><%= govuk_link_to("Read more about how schemes have changed", scheme_changes_path) %></p>
```

Comment:
missing a full stop ?

Reply:
Oh I think we don't want a full stop on either of those cause they're links